indexing-service/src/main/java/org/apache/druid/indexing/overlord/http/OverlordResource.java

```diff
@@ -562,6 +563,7 @@ public Response getTasks(
       @QueryParam("datasource") final String dataSource,
       @QueryParam("createdTimeInterval") final String createdTimeInterval,
       @QueryParam("max") final Integer maxCompletedTasks,
+      @QueryParam("maxActiveTasks") final Integer maxActiveTasks,
```

Comment:
I think we don't need to add this new parameter because there's already a parameter named as 'max'.  The 'max' should limit not only completedTasks but also activeTasks. When parameter 'state' is not given, it limits the total number of tasks.

Reply:
There're 2 items I'd like to support though:
1. It's more flexible and UI has more control over the backend.
2. It's backward compatible.